pythonx/UltiSnips/snippet/source/file/ultisnips.py

```diff
@@ -65,12 +65,20 @@ def _handle_snippet_or_global(filename, line, lines, python_globals, priority):
     # Get and strip options if they exist
     remain = line[len(snip):].strip()
     words = remain.split()
+
     if len(words) > 2:
         # second to last word ends with a quote
         if '"' not in words[-1] and words[-2][-1] == '"':
             opts = words[-1]
             remain = remain[:-len(opts) - 1].rstrip()
 
+    if 'x' in opts:
```

Comment:
Why did you choose 'x' as the option? Should this not be 'e' for expression? or 'p' for python?